src/native/http_connection_manager.c

```diff
@@ -96,30 +96,30 @@ JNIEXPORT jlong JNICALL Java_software_amazon_awssdk_crt_http_HttpClientConnectio
     struct aws_http_connection_manager *conn_manager = NULL;
 
     if (!client_bootstrap) {
-        aws_jni_throw_runtime_exception(env, "ClientBootstrap can't be null");
+        aws_jni_throw_illegal_argument_exception(env, "ClientBootstrap can't be null");
```

Comment:
```IllegalArgumentException```s should always contain the function name. I almost want to make it a separate argument to force it. Thoughts?